src/cdk/a11y/README.md

```diff
@@ -0,0 +1,30 @@
+# LiveAnnouncer
+`LiveAnnouncer` is a service, which announces messages to several screenreaders.
```

Comment:
These will all get revised when we write overview files for stuff in the cdk like the rest of the library.